### DISCUSSION TITLE
openSUSE Leap 15.4 Base

### DISCUSSION THREAD

**Discussion by Cameron-Cn** [uyuni-project/uyuni#5574] [Q&A] [answered]
Hello Geekos and friends!

I'm wondering where I would look to see about Uyuni running on Leap 15.4 as a base OS? I know Leap only just came out with the GM, and work is certainly underway; but does such a page exist for 'Next Steps'?

Thank you for your time and assistance!

**Comment by mcalmer** [comment, marked as answer]
Just wait for Uyuni 2022.06. It will switch the base

**Reply by Cameron-Cn** [reply]
Wonderful, thank you!   "Imminent" then seems to be the answer. :)
